[FEATURE] Internship Page

adityayadav71 opened the pull request:
1. Completed  #19  
2. Created internship.astro page in pages.
3. Created InternshipHero.astro, CompanyCard.tsx and InternshipCompanies.tsx component in src/Internships directory.
4. Used Array to map CompanyCard components in InternshipCompanies component.
5. Used CountUp component from Website Hero section in InternshipHero component.
6. Added necessary assets in public/InternshipHero directory.

adityayadav71 committed:
added assets for Internship page

adityayadav71 committed:
created internship.astro page in pages

adityayadav71 committed:
created internship hero component

adityayadav71 committed:
created internship companies component

adityayadav71 committed:
Merge branch 'staging' of https://github.com/tcet-opensource/tnp-website into 13-build-internship-page

adityayadav71 committed:
added bottom border  to title

adityayadav71 committed:
added CountUpComponent in InternshipHero

riyavishwa981 commented on src/components/Internships/InternshipHero.astro:
```diff
@@ -0,0 +1,27 @@
+    <div class="mb-6 rounded-xl">
+      <img src="/InternshipHero/img_1.png" class="rounded-xl" alt="image-of-a-female-student-pointing-towards-a-laptop-screen" />
+    </div>
+    <div class="flex sm:flex-row flex-col items-center w-full gap-3">
```
The two boxes are not aligning with the image ty to fix that

riyavishwa981 commented on src/components/Internships/InternshipHero.astro:
```diff
@@ -0,0 +1,27 @@
+    <div class="mb-6 rounded-xl">
+      <img src="/InternshipHero/img_1.png" class="rounded-xl" alt="image-of-a-female-student-pointing-towards-a-laptop-screen" />
+    </div>
+    <div class="flex sm:flex-row flex-col items-center w-full gap-3">
```
The two count boxes are not aligning with the image ty to fix that

adityayadav71 replied:
Fixed now

adityayadav71 committed:
fixed responsive image issue in InternshipHero

JayeshVP24 commented:
Hero section is very well done, nice job there
now the companies list part

- [x] 8 rows on 2xl feels very filled. make it 6
- [x] change text color of company name -> `text-slate-600` prefer to use tailwind defaults
- [x] change border below companies offering internship title to `border-slate-200`
- [x] also increase this borders width. it feels too small. make it responsive too. on large screens make the width shorter. and small screens make it longer. Try to match it with figma design

rest all is great. good use of reusable components and other folders components. great job

adityayadav71 committed:
changed text colors to use tailwind classes

adityayadav71 committed:
[FIX] border responsiveness and grid columns

adityayadav71 commented:
Thanks for reviewing my PR. I've made the requested changes:

- Reduced rows to 6 on 2xl screens
- Changed text color of company name to text-slate-600
- Changed border color and increased width below companies offering internship title
- Made the border responsive as per the Figma design
Appreciate your feedback and I am looking forward to your further review.

JayeshVP24 committed:
Merge branch 'staging' of https://github.com/tcet-opensource/tnp-website into 13-build-internship-page

JayeshVP24 committed:
[MERGE] with staging#112
